Sources/HummingbirdWebSocket/WebSocketHandler.swift

```diff
@@ -24,6 +24,14 @@ public enum WebSocketType: Sendable {
     case server
 }
 
+/// Automatic ping setup
+public enum AutoPingSetup: Sendable {
```

Comment:
Is this auto-ping setup a behaviour that could be expanded in the future? If so, we should un-freeze the API using a `struct`

Reply:
I doubt it'll need extending but made it into a struct anyway.